source/installguide/hypervisor/kvm.rst

```diff
@@ -645,11 +645,10 @@ although a reboot is recommended to see if everything works properly.
 Network Example for Advanced Networks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In the Advanced networking mode is most common to have (at least) two physical interfaces.
-In this example we will again have the hypervisor management interface on cloudbr0 on the 
-untagged (native) VLAN. But now we will have a bridge on top of our additional interface (eth1)
-for public and guest traffic with no VLANs applied by us - CloudStack will add the VLANs
-as required.
+In the Advanced networking mode is most common to have (at least) two physical interfaces per hypervior-host.
+We will use the interface eth0 linked to the bridge 'cloudbr0' using the untagged (native) VLAN for hypervisor management.
+Additionally we configure a the second interface for usage with the bridge 'cloudbr1' for public and guest traffic.
```

Comment:
`Additionally we configure a the second interface`
:arrow_down: I would suggest removing `a`, but you might remove `the` instead, if you prefer
`Additionally, we configure the second interface`

Reply:
Added suggestions  - thanks for checking and reading carefully :-)